script.js

```diff
@@ -169,9 +149,26 @@ const checkIfMatch = (event) => {
   }
 };
 
+const checkIfWon = () => {
+    const isWin = foundPairs === wordPairs.length;
+    const shouldSetupNextRound =
+      foundPairs % pairCount === 0 && foundPairs !== 0 && !isWin;
```

Comment:
Maybe lets split these even further: `foundPairs % pairCount === 0` and ` foundPairs !== 0` can go to separate variables